src/app/agent/agent-dashboard/agent-dashboard.component.ts

```diff
@@ -53,29 +60,24 @@ export class AgentDashboardComponent implements OnInit {
     this.adjustForMobile(window.innerWidth);
   }
 
+  ngOnDestroy() {
+    document.getElementById("nav-agent").classList.remove("active");
+  }
+
   // RESPONSIVE DESIGN
   @HostListener("window:resize", ["$event"])
   onResize(event) {
     this.adjustForMobile(event.target.innerWidth);
-  }
-
-  ngOnDestroy() {
-    document.getElementById("nav-agent").classList.remove("active");
-  }
+  } 
 
   adjustForMobile(width) {
     if(width < 992) {
       this.mobile = true;
-      this.moveSidebarToNav();
```

Comment:
Why is this no longer needed?

Reply:
It wasn't needed long ago, just forgot to remove it.